src/test/java/com/faforever/api/data/JsonApiClanIntegrationTest.java

```diff
@@ -88,146 +65,97 @@ public void setUp() {
         .webAppContextSetup(context)
         .addFilter(springSecurityFilterChain)
         .build();
-    me = null;
+    database.assertEmptyDatabase();
   }
 
-  // TODO @Dragonfire clean up the duplicated code
   @After
   public void tearDown() {
-    clanMembershipRepository.deleteAll();
-    clanRepository.deleteAll();
-    userRepository.deleteAll();
-    oAuthClientRepository.deleteAll();
-    assertEquals(0, clanMembershipRepository.count());
-    assertEquals(0, clanRepository.count());
-    assertEquals(0, userRepository.count());
-    assertEquals(0, oAuthClientRepository.count());
+    // TODO: This is needed, because Elide has some problems with @Transactional annotation
+    database.tearDown();
```

Comment:
Yes,  I (we?) will, but I will do it stepwise (iterative) and I know you don't like it, but I prefer sometimes working tests over no tests :p

I created an issue for this
https://github.com/FAForever/faf-java-api/issues/71